### PR TITLE
Basic Sign-up page 

### LINKED ISSUE #437: Add a basic sign-up page for clients
**Project title**
Basic sign-up page for client use

**Project Description**
A basic sign-up which takes some basic information from the user and let's them choose an username and password

**Stack**:  
- [x] Html
- [x] CSS

### PR DESCRIPTION
**Pull request by srijanishere**
<hr>

## Description 📜

This basic sign-up page takes in some information from the user and let's him/her select an username and password

Fixes #437 

<hr>

## Type of change 📝

- [x] Add a new project

<hr>

## Domain of Contribution 📊

- [x] Project using only HTML & CSS

<hr>
 
## Checklist ✅

Please delete options that are not relevant.And in order to tick the check box just but x inside them for example [x] like this

- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GSSoC'22 contributor

<hr>

### CODE REVIEW

**Review by meinhoonharsh — CHANGES_REQUESTED** [review]
I would be great if you can add a Screenshot/GIF or live link for the project
and I am not getting the use of `.DS_Store` file use here

**Comment by srijanishere** [comment]
> I would be great if you can add a Screenshot/GIF or live link for the project
> 
> and I am not getting the use of `.DS_Store` file use here

I will definitely make the above changes required, kindly allow me some time since I'm not in possession of my laptop right now. Thank you

**Comment by srijanishere** [comment]
@meinhoonharsh I've made the changes as per your request. Please re-review my changes and let me know if any further changes need to be made. Thank you.

File: Sign-up page/readme.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+4. Password
+
+## Tech Stack Used
+<span>
+  <img src="https://cdn-icons-png.flaticon.com/512/1051/1051277.png" width=40px height=40px> &nbsp; &nbsp;
+  <img src="https://cdn-icons-png.flaticon.com/512/732/732190.png" width=40px height=40px>
+</span>
+
+## Quick View
+<img src="https://github.com/srijanishere/WebDev-ProjectKart/blob/master/Sign-up%20page/Assets/Screenshot%202022-04-03%20at%2010.46.09%20AM.png">
```
Comment (meinhoonharsh) [inline Sign-up page/readme.md:17]: using some descriptive name for the image would be really nice

**Review by meinhoonharsh — CHANGES_REQUESTED** [review]
just one more small change and it is good to go

File: Sign-up page/readme.md (hunk 1 above)
Comment (srijanishere) [reply Sign-up page/readme.md]: I've changed the name of the image file with a descriptive name. Hope this will solve the issue.

**Review by meinhoonharsh — APPROVED** [review]
@khushi-purwar LGTM

**Review by khushi-purwar — APPROVED** [review]

**Pull request merged**